src/main/java/bg/startit/spring/firstspringproject/config/SecurityConfig.java

```diff
@@ -16,9 +15,10 @@
   @Override
   protected void configure(HttpSecurity http) throws Exception {
     http
+        .csrf()
+        /**/.disable()
```

Comment:
This is the actual fix to the problem of not being allowed to perform the request.